### ISSUE TITLE
Requirement.txt

### ISSUE THREAD

**Issue by pakkaponvstw** [magmagcup/django-polls#3]
Too many Requirement Django and Python is already too much.

**Comment by magmagcup** [comment]
No, it doesn't. My setting.py in mysite directory is using decouple library to separate my setting parameters from my source code so you must have decouple library in your python environment to run my poll application.
